frontend/test/metabase-smoketest/user.cy.spec.js

```diff
@@ -53,7 +51,7 @@ describe("smoketest > new_user", () => {
 
     cy.get(".TableInteractive-cellWrapper--firstColumn")
       .last()
-      .contains("Lightweight Steel Watch");
+      .findByText("Lightweight Steel Watch");
```

Comment:
I think we may want this to be: `.last().should('have.value', 'Lightweight Steel Watch')` to ensure that the last column is that value.

Reply:
On the frontend "Lightweight Steel Watch" isn't stored as the value of the cell; it's stored as the cell's text. Should it be stored as the value as well?